install/installer/pkg/common/common.go

```diff
@@ -339,7 +339,7 @@ func Affinity(orLabels ...string) *corev1.Affinity {
 	}
 }
 
-func RepoName(repo, name string) string {
+func RepoName(repo, name string, cfg *config.Config) string {
```

Comment:
@MrSimonEmms Have you thought about adding tests for this? I wonder if we could have tests that render some config, and then parse those YAML to validate that:
 - all image refs are valid
 - if airgapped is enabled: all image refs are fully qualified
 - whatever other variant there might be

Reply:
Alright, stupid me: It looks like there are [unit tests](https://github.com/gitpod-io/gitpod/pull/9155/files#diff-c7027dcc8c06120d3bd319bd1e23b4ed93915ffa83917b729cc091226dc72efaR47) already. :heavy_check_mark:

Reply:
🤣